arch/x86/soc/intel_quark/quark_se/soc.c

```diff
@@ -15,13 +15,15 @@
 #include <errno.h>
 
 #include <kernel.h>
-#include <misc/printk.h>
 #include <misc/__assert.h>
 #include "soc.h"
 #include <uart.h>
 #include <init.h>
 #include "shared_mem.h"
 #include <mmustructs.h>
+#include <logging/sys_log.h>
+#define SYS_LOG_LEVEL CONFIG_SYS_LOG_ARC_INIT_LEVEL
```

Comment:
SYS_LOG_LEVEL should be be defined before #including sys_log.h.  See logging/sys_log.h:24.

Reply:
yep, have fixed this locally already